src/scripts/fix_ws_metadata/fix_workspace_info.py

```diff
@@ -12,8 +12,8 @@
 import sys
```

Comment:
is this script still a going concern?

Reply:
No, I don't think so. We should put it somewhere for reference, but it doesn't need to be maintained.